samples/msgext-unfurling-ac-loop-components/nodejs/README.md

```diff
@@ -0,0 +1,140 @@
+---
+page_type: sample
+description: This sample showcases messaging extension unfurling adaptive card based loop components.
```

Comment:
This sample shows messaging extension with adaptive card-based loop components functionality.

Reply:
Fixed